Zend/zend_ast.c

```diff
@@ -654,6 +654,51 @@ ZEND_API int ZEND_FASTCALL zend_ast_evaluate(zval *result, zend_ast *ast, zend_c
 				zval_ptr_dtor_nogc(&op1);
 			}
 			break;
+		case ZEND_AST_MATCH:
```

Comment:
`zend_const_expr_to_zval` would also benefit from being updated to support ZEND_AST_MATCH, but it'd only matter if this were to be supported in constant expressions